### ISSUE TITLE
Doesn't Work with Tmux

### ISSUE THREAD

**Issue by Tanish2002** [karlstav/cava#339]
My `$TERM` variable with **tmux** is `tmux-256color` and with my **normal term** is `st-256color` 

Cava Isn't working with tmux but works with normal terminal(st)

Image with Tmux : H[ere](https://imgur.com/2x7FABn)

**Comment by karlstav** [comment]
@Tanish2002 just to make sure, does it work without user defined colors ( and without gradient)?

have you chekced #216? I think `$TERM` should be `rxvt-unicode-256color` or maybe `st-256color`

**Comment by Tanish2002** [comment]
It just doesn't wanna open with tmux... even if I use the basic config file, for now I changed tmux's $TERM var to st-256color and it works fine

**Comment by karlstav** [comment]
https://github.com/tmux/tmux/wiki/FAQ#how-do-i-use-rgb-colour

**Comment by Tanish2002** [comment]
still did not work
I've switched to suckless's tabbed now so you can close the issue if you want

**Comment by maujim** [comment]
I spent too much time yesterday trying to get cava to run (with gradients) inside tmux on st and I just wanted to leave this here in case others find this. 

The "solution" mentioned in the issue linked earlier (https://github.com/karlstav/cava/issues/216) is wrong as the `$TERM` should be one of `tmux, screen, tmux-256color, screen-256color` as indicated in the [tmux FAQ](https://github.com/tmux/tmux/wiki/FAQ).

Cava with gradients should work fine in st outside of tmux but inside it seems that the `tmux-256color` TERM will not support gradients. The best workaround I've found is the following alias
```
alias cava="TERM=st-256color cava"
```
